dokka-runners/dokka-gradle-plugin/build.gradle.kts

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+@file:Suppress("UnstableApiUsage") // jvm test suites
+
+import dokkabuild.tasks.GenerateDokkaGradlePluginConstants
+import dokkabuild.utils.skipTestFixturesPublications
+
+plugins {
+    id("dokkabuild.gradle-plugin")
+
+    kotlin("plugin.serialization") version embeddedKotlinVersion
+
+    id("dokkabuild.dev-maven-publish")
+    alias(libs.plugins.kotlinx.binaryCompatibilityValidator)
+    `java-test-fixtures`
+    `jvm-test-suite`
+
+    id("dokkabuild.publish-base")
+    alias(libs.plugins.gradlePluginPublish)
```

Comment:
probably this should be replaced with `id("dokkabuild.publish-gradle-plugin")` to configure `gradlePlugin` extension defaults and `javadoc` publication